src/@utils/url/index.test.ts

```diff
@@ -7,3 +7,15 @@ describe('@utils/url', () => {
     expect(sanitizeUrl('ftp://example.com')).toBe('about:blank')
   })
 })
+
+describe('isGoogleUrl', () => {
+  it('should return true if the url is a google domain', () => {
+    expect(isGoogleUrl('https://google.com')).toBe(true)
+    expect(isGoogleUrl('https://drive.google.com')).toBe(true)
+  })
+  it('should return false if the url is not a google domain', () => {
+    expect(isGoogleUrl('https://drive.gloogle.com')).toBe(false)
+    expect(isGoogleUrl('https://drive.google.test.com')).toBe(false)
+    expect(isGoogleUrl('https://google.com.test.com')).toBe(false)
```

Comment:
Can we also add some tests for other google services: google docs, google spreadsheet, etc.?

Reply:
Yup, done